micromamba/tests/test_install.py

```diff
@@ -741,3 +741,30 @@ def test_reinstall_with_new_version(tmp_home, tmp_root_prefix):
 
     res = helpers.umamba_run("-n", env_name, "python", "-c", "import pip; print(pip.__version__)")
     assert len(res)
+
+
+env_yaml_content_to_install_empty_base = """
+channels:
+- conda-forge
+dependencies:
+- python
+- xtensor
+"""
+
+
+def test_install_empty_base(tmp_home, tmp_root_prefix, tmp_path):
```

Comment:
Such a test was missing, i.e test creation of a valid environment with `install` when target prefix is the root prefix.